lib/components/app/desktop-nav.js

```diff
@@ -45,7 +45,7 @@ const DesktopNav = ({ otpConfig }) => {
       <Navbar.Header>
         <Navbar.Brand>
           {/* TODO: Reconcile CSS class and inline style. */}
-          <div className='app-menu-container' style={{ color: 'white', float: 'left', fontSize: 28 }}>
+          <div className='app-menu-container' style={{ color: 'white', float: 'left', fontSize: 28, marginTop: '5px' }}>
```

Comment:
Does `font-size` still apply? (the hamburger is made of `span`s).

Reply:
Removing it in 542d756098ceae9ad56f74334fae242b4be7c938 seemed to not affect things